app/models/miq_server.rb

```diff
@@ -625,7 +625,7 @@ def display_name
   end
 
   def server_timezone
-    get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
+    @timezone ||= get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
```

Comment:
I would rather get_config got changed to use the Settings instead of for_resource on a local server.  What I'm describing is also basically the fix to the bug you found about how we hit settings on every.  I'm only concerned here because we cache MiqServer objects.